### PR TITLE
Schema-less URLs

### PR DESCRIPTION
**Pull request by odahcam**
Closes pull:#137

### CODE REVIEW

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -84,13 +84,13 @@
 </main>
 
 <footer>
-    <span>&copy; Copyright <a href="http://thephpleague.com">The League of Extraordinary Packages</a>.</span>
-    <span>Site design by <a href="http://reinink.ca">Jonathan Reinink</a>.</span>
```
Comment (ragboyjr) [inline docs/_layouts/default.html]: Keep http.

File: docs/_layouts/default.html [hunk 2]
```diff
@@ -84,13 +84,13 @@
 </main>
 
 <footer>
-    <span>&copy; Copyright <a href="http://thephpleague.com">The League of Extraordinary Packages</a>.</span>
```
Comment (ragboyjr) [inline docs/_layouts/default.html]: Keep http.

File: docs/_layouts/default.html [hunk 3]
```diff
@@ -54,7 +54,7 @@
         <span class="name">{{ site.data.project.title }}</span>
         <span class="tagline">{{ site.data.project.tagline }}</span>
     </a>
-    <a href="http://thephpleague.com/" class="league">
```
Comment (ragboyjr) [inline docs/_layouts/default.html]: Keep http.

File: docs/_layouts/default.html [hunk 4]
```diff
@@ -35,8 +35,8 @@
 </a>
 
 <section class="all_packages">
-    <a href="http://thephpleague.com/">
```
Comment (ragboyjr) [inline docs/_layouts/default.html]: Keep http.

File: docs/_layouts/default.html (hunk 1 above)
Comment (odahcam) [reply docs/_layouts/default.html]: done.

File: docs/_layouts/default.html (hunk 2 above)
Comment (odahcam) [reply docs/_layouts/default.html]: done.

File: docs/_layouts/default.html (hunk 3 above)
Comment (odahcam) [reply docs/_layouts/default.html]: done.

File: docs/_layouts/default.html (hunk 4 above)
Comment (odahcam) [reply docs/_layouts/default.html]: done.

**Pull request merged**